lib/ansible/modules/windows/win_command.ps1

```diff
@@ -117,19 +117,21 @@ $exec_args = $null
 # FUTURE: no longer necessary once we switch to raw Win32 CreateProcess
 $parsed_command_line = [Ansible.Command.NativeUtil]::ParseCommandLine($raw_command_line);
 $exec_application = $parsed_command_line[0]
-If($parsed_command_line.Length -gt 1) {
-    # lop the application off, then rejoin the args as a single string
-    $exec_args = $parsed_command_line[1..$($parsed_command_line.Length-1)] -join " "
+If ($parsed_command_line.Length -gt 1) {
+    $exec_args = $parsed_command_line[1..-1]
```

Comment:
PS doesn't really have array slices, so go downvote that SO answer ;)- it accepts a list of indices in the brackets, which the range operator `..` can be used to generate in limited cases.